### PR TITLE
[BO-1917] Change version to epoch

### PR DESCRIPTION
**Pull request by patrickptasinskift**
Change version to epoch

### CODE REVIEW

File: lib/audited/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module Audited
-  VERSION = "4.9.4"
+  VERSION = "4.9.5"
```
Comment (mikalai-okun) [inline lib/audited/version.rb:2]: we should set a version number to be major, like 5.0.0-ft or something.

File: lib/audited/audit.rb [hunk 2]
```diff
@@ -178,8 +178,7 @@ def self.collection_cache_key(collection = all, *)
     private
 
     def set_version_number
-      max = self.class.namespaced.not_before_created_at(auditable).auditable_finder(auditable_id, auditable_type).maximum(:version) || 0
-      self.version = max + 1
+      self.version = DateTime.now.strftime('%Q').to_i
```
Comment (mikalai-okun) [inline lib/audited/audit.rb:181]: will that make an update SQL query?

Comment (patrickptasinskift) [reply lib/audited/audit.rb:181]: Not in this line of code, but overall it will be only inserts into database

File: lib/audited/version.rb (hunk 1 above)
Comment (patrickptasinskift) [reply lib/audited/version.rb:2]: If we decide to fix tests then yeah, I agree

**Comment by mikalai-okun** [comment]
@patrickptasinskift test are failing

**Comment by patrickptasinskift** [comment]
> @patrickptasinskift test are failing

@mikalai-okun To fix the tests, we would need to do major refactor of this gem. For now I guess we can stay with patching through initializers and close this PR?

**Comment by mikalai-okun** [comment]
Sure, please provide a patch

**Pull request closed without merging**